### PR TITLE
Fixed icon docs

### PR DESCRIPTION
**Pull request by galgalesh**
The docs for the logo have been out of date for a long time. This PR fixes the docs to resemble the circular icon style now used in the charm store.

### CODE REVIEW

**Comment by galgalesh** [comment]
fixes https://github.com/juju/docs/issues/1187

**Comment by pmatulis** [comment]
@evilnick You seem to know something about this stuff.

File: src/en/authors-charm-icon.md [hunk 1]
```diff
@@ -2,36 +2,22 @@ Title: Creating icons for charms
 
 # Creating icons for charms
 
-In order to make your charm recogniseable in the Charm Store, it is best
-practice to add a unique and recogniseable icon for it. This isn't as hard as it
-may seem, as this step by step guide will demonstrate:
+In order to make your charm recognizable in the Charm Store, it is best
```
Comment (evilnick) [inline src/en/authors-charm-icon.md]: That was a typo, but it should be "recognisable"

File: src/en/authors-charm-icon.md [hunk 2]
```diff
@@ -40,61 +26,37 @@ first place, then fear not, because we have an easy step-by-step guide for you.
...
-showing the limits of the pictogram area.
-
-![Step three](./media/author-charm-icons-3.png)
-
 ## Draw something
 
-Draw your shape within the bounds determined by the guides. If you already have
-a vector logo, you can import it and scale it within the guides. Inkscape also
-has plenty of drawing tools for creating complex images.
+Draw your shape within the circle If you already have a vector logo, you can
```
Comment (evilnick) [inline src/en/authors-charm-icon.md]: needs a '.' after circle

File: src/en/authors-charm-icon.md (hunk 1 above)
Comment (galgalesh) [reply src/en/authors-charm-icon.md]: afaik, both are correct, it's US vs UK English. My spell checker must be on US English..

**Comment by galgalesh** [comment]
Fixed

File: src/en/authors-charm-icon.md (hunk 1 above)
Comment (evilnick) [reply src/en/authors-charm-icon.md]: Yes, the 'z' spelling would be US spelling. All the docs are supposed to be UK English, though I think a few zs have slipped through. Thanks again!

**Pull request merged**